modules/github/README.md

```diff
@@ -15,6 +15,7 @@ Config explanations:
       - credentials:
             - oauthtoken: a Github OAuth2 or Personal Access Token
       - parameters:
+            - useldapgithubaccountmapping: indicates whether to use the Github field in LDAP as opposed to the uidmap in config to map LDAP usernames to Github usernames
```

Comment:
I'm not sure we should do an either/or behavior here. Userplex is an LDAP-to-many multiplexer, and I think we should require the data to be in LDAP. If a users don't have their github usernames in LDAP, we should just remove them from the org.

Given this, I'm not sure we need to use the uidmap at all in this module.